src/context.tsx

```diff
@@ -43,4 +39,48 @@ const init = function(){
     if (storage) state.fallenHeroes= JSON.parse(storage)
 }
 init()
-console.log(state.fallenHeroes, state.restOfAliveHeroes)
\ No newline at end of file
+console.log(state.fallenHeroes, state.restOfAliveHeroes)
+
+*/
+
+export interface CampaignContextInterface {
+  campaign: Campaign;
+  setCampaign: Dispatch<SetStateAction<Campaign>>;
+}
+
+const defaultState = {
+  campaign: {
+    fallenHeroes: [],
+    restOfAliveHeroes: ["Aragorn", "Eowyn", "Beravor"],
+    allHeroes: ["Aragorn", "Eowyn", "Beravor"],
+  },
+  setCampaign: (campaign: Campaign) => {},
+} as CampaignContextInterface;
+
+const LOTRContext = React.createContext(defaultState);
+
+type CampaignProvideProps = {
+  children: ReactNode;
+};
+const LOTRProvider = ({ children }: CampaignProvideProps) => {
```

Comment:
Proper PascalCase name would be "LotrProvider".

Reply:
changed to PascalCase